### PR TITLE
[Lua] Fix several Cryptonberry (CoP 7-4 mobs) issues

### PR DESCRIPTION
**Pull request by TracentEden2**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?
This PR fixes several issues with Cryptonberry mobs:
1. Changes the 2hr logic of the mobs to follow the custom from retail captures
2. Add the correct status immunities
3. Change the 2hrs of the Cryptonberry Assassins to the correct 2hrs (for their jobs)
4. Add several missing mob texts

The changes are based on the retail captures by Siknoz [here](https://youtu.be/u9APAn_F5TU), zach2good [here](https://drive.google.com/open?id=157JfVoVzba4f_4DV1-q9GYV0pewuFkpH), and Jimmayus [here](https://youtu.be/sUXzgqsAr_g).

## Steps to test these changes
Fight the mobs and compare to the behavior from the retail captures.

### CODE REVIEW

File: scripts/zones/Carpenters_Landing/mobs/Cryptonberry_Assassin.lua [hunk 1]
```diff
@@ -4,36 +4,65 @@
...
-                    mobArg:messageText(mobArg, ID.text.CRYPTONBERRY_ASSASSIN_2HR)
-                end,
-            },
-        },
-    })
-end
-
-entity.onMobEngage = function(mob, target)
+    mob:addImmunity(xi.immunity.LIGHT_SLEEP)
+    mob:setMod(xi.mod.SLEEPRES, 90)
```
Comment (Xaver-DaRed) [inline scripts/zones/Carpenters_Landing/mobs/Cryptonberry_Assassin.lua]: Similar to what I posted in your previous PR. SLEEP_MEVA should be used here.

File: scripts/zones/Carpenters_Landing/mobs/Cryptonberry_Assassin.lua [hunk 2]
```diff
@@ -4,36 +4,65 @@
...
+        mob:setLocalVar('usedTwoHour', 1)
+        mob:messageText(mob, ID.text.CRYPTONBERRY_ASSASSIN_2HR)
+        mob:useMobAbility(cryptonberrySpecials[mob:getMainJob()])
+    end
 end
 
 entity.onMobDeath = function(mob, player, optParams)
+    if
+        mob:getLocalVar('firstOnMobDeathCall') == 0
+    then
```
Comment (Xaver-DaRed) [inline scripts/zones/Carpenters_Landing/mobs/Cryptonberry_Assassin.lua]: 1 line
`if mob:getLocalVar('firstOnMobDeathCall') == 0 then`

File: scripts/zones/Carpenters_Landing/mobs/Cryptonberry_Executor.lua [hunk 3]
```diff
@@ -4,43 +4,84 @@
...
+    mob:setMod(xi.mod.REGAIN, 100)
 end
 
 entity.onMobFight = function(mob, target)
-    -- spawn Assassins when enmity is gained against Executor
-    if mob:getLocalVar('spawnedAssassins') == 0 and mob:getCE(target) > 0 then
+    -- spawn assassins when enmity is gained against Executor
+    if
+        mob:getLocalVar('spawnedAssassins') == 0 and mob:getCE(target) > 0
+    then
```
Comment (Xaver-DaRed) [inline scripts/zones/Carpenters_Landing/mobs/Cryptonberry_Executor.lua:39]: 4 lines
```lua
    if
        mob:getLocalVar('spawnedAssassins') == 0 and
        mob:getCE(target) > 0
    then
```

File: scripts/zones/Carpenters_Landing/mobs/Cryptonberry_Executor.lua [hunk 4]
```diff
@@ -4,43 +4,84 @@
...
+        mob:useMobAbility(731)
+        -- unlock the 2hrs of the assassins
+        unlockAssassinTwoHours()
+    end
 end
 
 entity.onMobDeath = function(mob, player, optParams)
+    if
+        mob:getLocalVar('firstOnMobDeathCall') == 0
+    then
```
Comment (Xaver-DaRed) [inline scripts/zones/Carpenters_Landing/mobs/Cryptonberry_Executor.lua]: 1 line
`if mob:getLocalVar('firstOnMobDeathCall') == 0 then`

File: scripts/zones/Carpenters_Landing/mobs/Cryptonberry_Assassin.lua (hunk 1 above)
Comment (TracentEden2) [reply scripts/zones/Carpenters_Landing/mobs/Cryptonberry_Assassin.lua]: > Similar to what I posted in your previous PR. SLEEP_MEVA should be used here.

Thanks for info, I have updated the PR!

File: scripts/zones/Carpenters_Landing/mobs/Cryptonberry_Assassin.lua (hunk 2 above)
Comment (TracentEden2) [reply scripts/zones/Carpenters_Landing/mobs/Cryptonberry_Assassin.lua]: > 1 line `if mob:getLocalVar('firstOnMobDeathCall') == 0 then`

Thanks, I have updated the PR!

File: scripts/zones/Carpenters_Landing/mobs/Cryptonberry_Executor.lua (hunk 3 above)
Comment (TracentEden2) [reply scripts/zones/Carpenters_Landing/mobs/Cryptonberry_Executor.lua:39]: > 4 lines
> 
> ```lua
>     if
>         mob:getLocalVar('spawnedAssassins') == 0 and
>         mob:getCE(target) > 0
>     then
> ```

Thanks, I have updated the PR!

File: scripts/zones/Carpenters_Landing/mobs/Cryptonberry_Executor.lua (hunk 4 above)
Comment (TracentEden2) [reply scripts/zones/Carpenters_Landing/mobs/Cryptonberry_Executor.lua]: > 1 line `if mob:getLocalVar('firstOnMobDeathCall') == 0 then`

Thanks, I have updated the PR!

**Review by Xaver-DaRed — APPROVED** [review]

**Pull request merged**